components/kyma-environment-broker/internal/provisioner/client.go

```diff
@@ -25,16 +30,34 @@ type Client interface {
 }
 
 type client struct {
-	graphQLClient *graphql.Client
+	graphQLClient *gcli.Client
 	queryProvider queryProvider
 	graphqlizer   Graphqlizer
+
+	queryDumping bool
 }
 
-func NewProvisionerClient(endpoint string, queryLogging bool) Client {
+func NewProvisionerClient(endpoint string, queryDumping bool) Client {
+	httpClient := &http.Client{
+		Transport: &http.Transport{
+			TLSClientConfig: &tls.Config{
+				InsecureSkipVerify: true,
+			},
+		},
+	}
+
+	graphQlClient := gcli.NewClient(endpoint, gcli.WithHTTPClient(httpClient))
+	if queryDumping {
+		graphQlClient.Log = func(s string) {
+			fmt.Println(s)
```

Comment:
can we use logger here? We already have a 2 approaches for dumping info to stdout, adding 3rd one is not a good idea IMO

Reply:
My idea is to dump the request as it is - it is more readable. It is hard to see in the log what is going to the Provisioner. Anyway, we should not log the request because of sensitive data there, so in short time we need to change it/disable. I've create an issue: https://github.com/kyma-incubator/compass/issues/1169

If you still don't like to dump directly to stdout, I can revert this change.

Reply:
If you created an issue, and we will address that in the near future then it is ok for me 👍